ios/browser/api/ads/brave_ads.mm

```diff
@@ -1568,6 +1584,34 @@ - (bool)hasPrefPath:(const std::string&)path {
   return [self.prefs objectForKey:key] != nil;
 }
 
+- (void)setLocalStatePref:(const std::string&)path value:(base::Value)value {
+  std::string json;
+  if (base::JSONWriter::Write(value, &json)) {
```

Comment:
nit: fail fast, so that we remove the indentation

Reply:
Converted to an empty function because we don't support this functionality for iOS.